### PR TITLE
Replace slanger with peatio-core ranger

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

**Comment by gsilvestri** [comment]
Is Ranger a Rubykube version of Slanger??

**Comment by AboShanab** [comment]
@gsilvestri It's the open-source alternative of Slanger.

**Comment by gsilvestri** [comment]
I see: can we replace slanger with ranger in our local peation installations??

**Comment by gsilvestri** [comment]
What I meant with my previous question @shal is: is there a public repository where I can download ranger currently??

rubykube/ranger repository is currently empty...
Ranger URL: https://github.com/rubykube/ranger

I do understand that this PR is about integrating Ranger with Peatio... BUT there is a repository where we can download Ranger now and integrate Ranger ourself with Peatio in our local repositories??

Thank You for your work and _have a productive day_

**Comment by kovetskiy** [comment]
@gsilvestri Hi, sorry that we missed your question.

Ranger is microservice that written in Ruby, it's provided by peatio gem that you can find in following repository: https://github.com/rubykube/peatio-core

The ranger is pretty simple: https://github.com/rubykube/peatio-core/blob/master/lib/peatio/ranger.rb
The docs are not merged yet, but you can take a look at them here: https://github.com/rubykube/peatio/blob/e20f470a6774bdc43404c333a3fd86f453259f27/docs/api/websocket_api.md

Feel free to ask any questions.

File: compose/daemons.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ services:
     << : *peatio-daemon
     command: bash -c "bin/link_config && bundle exec ruby lib/daemons/blockchain.rb"
 
+  blockchain:
```
Comment (kovetskiy) [inline compose/daemons.yaml]: blockchain -> global_state

**Pull request merged**